test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -905,6 +905,41 @@ try
         }
     }
 
+    Describe "New-Object constrained language test" -Tags "CI" {
+
+        It "New-Object object cannot create COM object under constrained mode" -skip:(-not $IsWindows) {

```

Comment:
Can this be another test case in the same context  https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1#L277